src/components/TopBar.tsx

```diff
@@ -62,45 +101,105 @@ interface Props {
   button?: React.ReactNode
 }
 
-const progressStrings = [
+const progressInfo = [
   {
     key: 0,
-    progressText: '1. Berätta om dig själv',
+    progressPercentage: '30%',
+    textKey: 'PROGRESS_TEXT_ONE',
   },
   {
     key: 1,
-    progressText: '2. Se din försäkring',
+    progressPercentage: '50%',
+    textKey: 'PROGRESS_TEXT_TWO',
```

Comment:
i think these text keys should be what they represent over what order they come in (i.e. `PROGRESS_TEXT_OFFER`)